lastuser_oauth/views/login.py

```diff
@@ -57,9 +55,7 @@
 
 def openid_log(message, level=0):
```

Comment:
Mark this function for deprecation with Flask-OAuth. As that module is only used for Twitter login, it should be replaced with Tweepy (separate PR).